app/user_manager.py

```diff
@@ -139,6 +140,9 @@ def get_user_data_path(request, check_exists = False, param = "file"):
             if not file:
                 return web.Response(status=400)
                 
+            # Decode URL-encoded file path
+            file = urllib.parse.unquote(file);
```

Comment:
You don't need that semicolon in python. :)
I would remove that.

```suggestion
            file = urllib.parse.unquote(file)
```

Reply:
@TobTobXX not sure why you marked this as resolved but didn't actually resolve it? There's still a semicolon there